### PR TITLE
OPENEUROPA-2340: Implement site specific footer for v1.x

### PR DESCRIPTION
**Pull request by sergepavle**
### Description

The scope of this ticket is to implement the site specific footer as outlined in OPENEUROPA-2339.

### Change log

- Added:
- Changed:
- Deprecated:
- Removed:
- Fixed:
- Security:

### Commands

```sh
[Insert commands here]

```

### CODE REVIEW

**Review by imanoleguskiza — CHANGES_REQUESTED** [review]

File: templates/oe-corporate-blocks-footer.html.twig [hunk 1]
```diff
@@ -4,18 +4,50 @@
...
 #}
 
-<div class="footer__corporate-top">
+{%  if custom_footer is not empty %}
+  <div class="footer__custom">
+    {%  if custom_footer.identity is not empty %}
+      <h2 class="footer__custom-title-identity">{{ custom_footer.identity }}</h2>
+    {% endif %}
+    {%  if custom_footer.social_links is not empty %}
+      <p class="footer__custom-label">Follow us:</p>
```
Comment (imanoleguskiza) [inline templates/oe-corporate-blocks-footer.html.twig]: This needs to be translatable.

**Review by imanoleguskiza — CHANGES_REQUESTED** [review]

File: tests/Behat/CorporateBlocksContext.php [hunk 2]
```diff
@@ -87,6 +87,25 @@ public function assertCorporateBlockLinksUrls(string $region, TableNode $links):
     }
   }
 
+  /**
+   * Assertion that there are no links in region.
+   *
+   * @Then Links in the :region region do not contains the links:
```
[the remark this answers is not in the document]
Comment (sergepavle) [reply tests/Behat/CorporateBlocksContext.php]: it is modified this step: `@Then Links in the :region region contains the links:`. See: `\Drupal\Tests\oe_corporate_blocks\Behat\CorporateBlocksContext::assertCorporateBlockLinksUrls`. Should I rename also this one?

File: src/Plugin/Block/FooterBlock.php [hunk 3]
```diff
@@ -81,9 +93,59 @@ public function build() {
...
+      foreach ($links_storage->loadMultiple($link_ids) as $link_entity) {
+        $cache->addCacheableDependency($link_entity);
+
+        $link = [
+          'href' => $link_entity->getUrl()->toString(),
+          'label' => $link_entity->label(),
+        ];
+
+        if ($type === 'footer_social_link') {
+          $link['social_network'] = $link_entity->getSocialNetwork();
```
[the remark this answers is not in the document]
Comment (sergepavle) [reply src/Plugin/Block/FooterBlock.php]: Sorry, I didn't get what is the alternative approach?

**Review by imanoleguskiza — APPROVED** [review]

**Comment by ademarco** [comment]
We won't provide this functionality on 1.x version. Closing.

**Pull request closed without merging**